### DISCUSSION TITLE
Count providers per active podcasts and not per episodes

### DISCUSSION THREAD

**Discussion by LaCollision** [skymethod/livewire-web#2] [Ideas]
Hi,

First, thank you very much for your very useful tool :)

I would just add my 2 cents.
Don't you think it would be more interesting to count active podcasts (= those with at least one episode published), rather than the number of episodes published?

Let's take an example :
• Host A has 3 very active podcasts (for instance web radios), each publishing 5 episodes per day.
That's over 30 days: 3 x 5 x 30 = 450 episodes / month.

• Host B has many more podcasts, for example 100, but each with a « normal » publication rate of one episode per week (ie 4 / month).
That's over 1 month: 100 x 4 = 400 episodes / month.

With the current ranking, host A will be ranked higher than host B.
However, host B has 100 active podcasts, while host A only has 3.

Also, it seems to me that it would be more interesting to count active podcasts rather than active episodes.

What do you think?

Thank you!

**Comment by johnspurlock-skymethod** [comment]
Hi there - it's a good idea.  

My current episode share analysis was born out of having this interesting new data set that I've not seen elsewhere, i.e. all unique new current episodes published in a month.  No sampling or guesswork, it uses the public observable data that exists and checks all of the media files - so tries to be as objective as possible.  Sometimes I think of it like viewing the output of a huge content production engine: if the engine stops - podcasting is dead, any contribution to creating new episodes: keeps the engine alive.  Like a heartbeat.

I'd like to eventually also do a "minute share" as well, since the contribution to the "new podcast content" universe is perhaps different for a 5 minute show vs a 3 hr show.

And yes, having another analysis for "top hosts by number of shows with at least one episode published in the last month" would be an interesting metric as well.  One thing I'll need to watch out for in that analysis would be duplicate / customized / aggregate feeds.  In other words: what is a show?  My current episode universe avoids this problem since it only uses unique media files, which may reside in more than one feed.

But yes, I would like to do a show-based analysis as well!  Adding to a long todo list : )  Keep an eye out in future months!
